src/nightsheet.json

```diff
@@ -73,7 +74,11 @@
   ],
   "otherNight": [
     "DUSK",
-    "cannibal",
```

Comment:
Cannibal was added to the Other Night order as a reminder for Storytellers that often forget to wake the Cannibal. Removing it would just reintroduce the forgetfulness issue that the original pull request aimed to mitigate.

Reply:
I prefer to have it, I skipped it too many times.